server/app/controllers/checks_controller.py

```diff
@@ -153,6 +164,8 @@ def run(check_id):
             raise ValueError(
                 "Check with id {} is not present.".format(check_id))
 
+    
+    # This method returns a list of all the checks based on the course_id.
```

Comment:
Does that mean it returns a list of all checks that pertain to a particular course_id, or that it returns a list of checks, ordered by course_id?

Reply:
That method returns a list of all checks for a particular `course_id`.  I'll rephrase the comment.